.gitignore

```diff
@@ -0,0 +1,58 @@
+# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
+
+# C extensions
+*.so
+
+# Distribution / packaging
+.Python
+env/
+build/
+develop-eggs/
+dist/
+downloads/
+eggs/
+lib/
+lib64/
+parts/
+sdist/
+var/
+*.egg-info/
+.installed.cfg
+*.egg
+
+# PyInstaller
+#  Usually these files are written by a python script from a template
+#  before PyInstaller builds the exe, so as to inject date/other infos into it.
+*.manifest
+*.spec
+
+# Installer logs
+pip-log.txt
+pip-delete-this-directory.txt
+
+# Unit test / coverage reports
+htmlcov/
+.tox/
+.coverage
+.cache
+nosetests.xml
+coverage.xml
+
+# Translations
+*.mo
+*.pot
+
+# Django stuff:
+*.log
+db.sqlite
+
+# Sphinx documentation
+docs/_build/
+
+# PyBuilder
+target/
+
+node_modules/
+
```

Comment:
I usually side with it being the individual developer's responsibility to add these things to their [global excludes](http://git-scm.com/docs/gitignore) file, which caters to their specific OS and tooling (since I tend to think of versioned gitignores as only encompassing project-specific generated files, rather than every possible generated file produced by every possible environment).

> Patterns which a user wants Git to ignore in all situations (e.g., backup or temporary files generated by the user’s editor of choice) generally go into a file specified by core.excludesFile in the user’s ~/.gitconfig.

That being said, I don't actually have a really strong preference on this. :wink: